src/content/concepts/hot-module-replacement.md

```diff
@@ -64,7 +64,7 @@ A `check` makes an HTTP request to the update manifest. If this request fails, t
 
 The `apply` method flags all updated modules as invalid. For each invalid module, there needs to be an update handler in the module or in its parent(s). Otherwise, the invalid flag bubbles up and invalidates parent(s) as well. Each bubble continues until the app's entry point or a module with an update handler is reached (whichever comes first). If it bubbles up from an entry point, the process fails.
 
-Afterwards, all invalid modules are disposed (via the dispose handler) and unloaded. The current hash is then updated and all `accept` handlers are called. The runtime switches back to the `idle` state and everything continues as normal.
+Afterwards, all invalid modules are disposed (via the dispose handler) and unloaded. The current hash is then updated and all `accept` handlers are called. The runtime switches back to the `idle` state and everything continue as normal.
```

Comment:
This change is not needed. `... everything continues` / `... everything will continue` are grammatically correct options.

Reply:
@superburrito ok